docs/upgrade_to_1_5.md

```diff
@@ -105,6 +105,10 @@ cast send \
     --rpc-url "$RPC_URL"
 ```
 
+> [!NOTE]
+> `cast send` will fail if the `History` type is not known to `cast` at the time the of the execution.
```

Comment:
At the end there's one too many `the`

`at the time of the execution.`

Reply:
Done at 10cd9ce9e0878237bfa9f7b940d4ec30f3c50052